### PR TITLE
Added Donk to validated-tokens

### PR DESCRIPTION
**Pull request by ceedeepee**
# Validate [DONK](https://solscan.io/token/xABfKiG2KCHi6keTeLycW1iK7B52wJmchSWXu3YrsDp)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/DonkOnSol/status/1724424711419883921
- Coingecko/ CMC URL (If available):  https://www.geckoterminal.com/solana/pools/4s54RyhmkAjZ5uCfQKMpdCLwR4JLMQPaVZbPNeYHfavY

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

File: validated-tokens.csv
Comment (00Smurf) [inline validated-tokens.csv:1]: [x] Symbol: DONK
[x] Token decimals: 5
[x] Token icon: https://arweave.net/yu68b5mpxYjDqL990DEL9b_ri9E8TX-o27E1PS9W6II
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by 00Smurf — COMMENTED** [review]
[x] Symbol: DONK
[x] Token decimals: 5
[x] Token icon: https://arweave.net/yu68b5mpxYjDqL990DEL9b_ri9E8TX-o27E1PS9W6II
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by tribixbite — COMMENTED** [review]
[x] Symbol: DONK
[x] Token decimals: 5
[x] Token icon: https://arweave.net/yu68b5mpxYjDqL990DEL9b_ri9E8TX-o27E1PS9W6II
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Comment by kingdigii** [comment]
[x] Symbol: DONK
[x] Token decimals: 5
[x] Token icon: https://arweave.net/yu68b5mpxYjDqL990DEL9b_ri9E8TX-o27E1PS9W6II
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by meijiao — DISMISSED** [review]

**Review by Timoon21 — DISMISSED** [review]

**Review by Timoon21 — DISMISSED** [review]

**Comment by Timoon21** [comment]
Plz read the warning message from github:

The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)

**Pull request merged**